testing/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -71,14 +157,14 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
     }
 
     int getLibJarsCount() {
-        coreLibJarsCount + packagedPluginsJarCount + agentJarsCount + thirdPartyLibJarsCount
+        coreLibJarsCount - CORRECT_FOR_JARS_THAT_ARE_ALREADY_IN_OTHER_COUNTS + packagedPluginsJarCount + agentJarsCount + thirdPartyLibJarsCount
```

Comment:
❓ Do we really need to make a correction?

This PR does not add or remove any projects, so I don't expect the jar count to change.
However, we increased the `getCoreLibJarsCount` by 2 and then compensated by the same 2 here, which seems redundant.

Shouldn't we just keep the numbers the same?

Reply:
I was able to avoid this but is still not clear to me how these counts are working.

The `packagedPluginsJarCount` is one more than the actually "gradle-*" count in plugins.

Reply:
fixed in other PR